libs/node_modules/ua-parser-js/package.js

```diff
@@ -0,0 +1,12 @@
+Package.describe({
```

Comment:
Is it intended that the sources of dependencies (here `ua-parser-js`) are checked in under `node_modules`?

Reply:
nope, thank you for noticing!